lux/interestingness/interestingness.py

```diff
@@ -127,8 +127,25 @@ def deviationFromOverall(view:View,ldf:LuxDataFrame,filterSpecs:list,msrAttribut
 	assert len(v) == len(v_filter), "Data for filtered and unfiltered view have unequal length." 
 	sig = v_filter_size/v_size #significance factor
 	# Euclidean distance as L2 function
+
+	rankSig = 1 #category measure value ranking significance factor
+	#if the view is a barchart, count how many categories' rank, based on measure value, changes after the filter is applied
+	if view.mark == "bar":
+		dimList = view.getAttrByDataModel("dimension")
+
+		#use Pandas rank function to calculate rank positions for each category
+		v_rank = unfilteredView.data.toPandas().rank().toPandas()
```

Comment:
Is there a specific reason why we need to do `toPandas` here? Can we not apply `unfilteredView.data.rank()` directly, since LuxDataFrame inherits from Pandas?

Reply:
Yeah we can just apply .rank() directly, missed that there.

Reply:
That sounds good, you can just make additional commits and push to your branch after you've removed the toPandas. It should automatically reflect on this pull request.